### PR TITLE
LPS-139091

### PR DESCRIPTION
**Pull request by linolaoj**

### CODE REVIEW

**Comment by rodrigopaulino** [comment]
ci:test:sf

**Comment by rodrigopaulino** [comment]
ci:test:relevant

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-item-selector-api/src/main/java/com/liferay/dynamic/data/mapping/form/item/selector/criterion/DDMUserPersonalFolderItemSelectorCriterion.java [hunk 1]
```diff
@@ -36,6 +36,10 @@ public long getFolderId() {
 		return _folderId;
 	}
 
+	public long getFormInstanceId() {
+		return _formInstanceId;
+	}
+
```
Comment (carolmariaabb) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-item-selector-api/src/main/java/com/liferay/dynamic/data/mapping/form/item/selector/criterion/DDMUserPersonalFolderItemSelectorCriterion.java:42]: @linolaoj can you rename the methods to `getDDMFormInstanceId` and `setDDMFormInstanceId` and the variable to `_ddmFormInstanceId`? Thanks

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/document/library/DocumentLibraryDDMFormFieldTemplateContextContributor.java [hunk 2]
```diff
@@ -451,6 +451,9 @@ private String _getItemSelectorURL(
 				new DDMUserPersonalFolderItemSelectorCriterion(
 					folderId, groupId);
 
+		ddmUserPersonalFolderItemSelectorCriterion.setFormInstanceId(
+			ddmFormFieldRenderingContext.getDDMFormInstanceId());
+
 		ddmUserPersonalFolderItemSelectorCriterion.
 			setDesiredItemSelectorReturnTypes(
 				new FileEntryItemSelectorReturnType());
```
Comment (carolmariaabb) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/document/library/DocumentLibraryDDMFormFieldTemplateContextContributor.java:459]: @linolaoj can you sort this alphabetically and remove the blank line? Thanks

**Review by carolmariaabb — CHANGES_REQUESTED** [review]

**Comment by linolaoj** [comment]
Hi @carolmariaabb , I've pushed the changes. Please let me know if any other change is needed.
Thanks!

**Comment by linolaoj** [comment]
ci:test:relevant

**Comment by carolmariaabb** [comment]
ci:reevaluate:1277401_388

**Comment by carolmariaabb** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
Hey @linolaoj, can you please check if these failures are related to your changes? Thanks

**Comment by carolmariaabb** [comment]
Hi @linolaoj,
The CI failures seem to be related to your changes.  Can you please fix them and resend the PR? Feel free to contact us if you need any help with this.


image

**Pull request closed without merging**